src/content/en/android/trusted-web-activity/quick-start/index.md

```diff
@@ -49,32 +50,75 @@ out, this is called **verification**.
 If verification fails, the browser will fall back to displaying your website as a
 [Custom Tab](https://developer.chrome.com/multidevice/android/customtabs).
 
-## Clone and customize the example repo {: #clone-and-customize-the-example-repo }
+## Install and configure Bubblewrap {: #install-and-configure-bubblewrap }
 
-The [svgomg-twa](https://github.com/GoogleChromeLabs/svgomg-twa) repo contains an example
-Android application that launches a Trusted Web Activity. You can customize to launch your website:
+[Bubblewrap](https://github.com/GoogleChromeLabs/bubblewrap) is a set of libraries and a command
+line tool (CLI) for Node.js that helps developers generate, build and run Progressive Web Apps
+inside Android applications, using Trusted Web Activity.
 
-1. Clone the project (`git clone https://github.com/GoogleChromeLabs/svgomg-twa.git`).
-2. Import the Project into Android Studio, using **File** > **New** > **Import Project**, and select
-   the folder to which the project was cloned.
-3. Open the app's `build.gradle` and modify the values in `twaManifest`.
-   There are two `build.gradle` files.
-   You want the module one at `app/build.gradle`.
+The CLI can be installed with the following command:
 
-    * Change `hostName` to point to your website.
-      Your website must be available on HTTPS, though you omit that from the `hostName` field.
-    * Change `name` to whatever you want.
-    * Change `applicationId` to something specific to your project.
-      This translates into the app’s package name and is
-      [how the app is identified](https://developer.android.com/studio/build/application-id) on the
-      Play Store - no two apps can share the `applicationId` and if you change it you’ll need to
-      create a new Play Store listing.
+```shell
+npm i -g @bubblewrap/cli
+```
+
+### Setting up the Environment
+
+#### Get the Java Development Kit (JDK) 8.
+The Android Command line tools requires the correct version of the JDK to run. To prevent version
+conflicts with a JDK version that is already installed, Bubblewrap uses a JDK that can unzipped in
+a separate folder.
+
+Download a version of JDK 8 that is compatible with your OS from
+[AdoptOpenJDK](https://adoptopenjdk.net/releases.html?variant=openjdk8&jvmVariant=hotspot)
+and extract it in its own folder.
+
+**Warning:** Using a version lower than 8 will make it impossible to compile the project and higher
```

Comment:
I'd put the warning above the download link, just so people don't read the download link, follow it, pick a different version, extract it and *then* read your warning.

Reply:
done.